service_management/app/repositories/loan/dao/Tables.scala

```diff
@@ -0,0 +1,152 @@
+package repositories.loan.dao
+
+import java.sql.{Timestamp => JSqlTimestamp}
+
+import models.loan.LoanEventTypes.{ObjectLentType, ObjectReturnedType}
+import models.loan.LoanType
+import models.loan.event.{LoanEvent, ObjectsLent, ObjectsReturned}
+import no.uio.musit.models._
+import no.uio.musit.time.Implicits._
+import org.joda.time.DateTime
+import play.api.db.slick.HasDatabaseConfigProvider
+import play.api.libs.json.{JsValue, Json}
+import repositories.shared.dao.ColumnTypeMappers
+import slick.jdbc.JdbcProfile
+
+/**
+ * Defines the table definitions needed to work with analysis data.
+ * Also includes a few helpful functions to convert between rows and types.
+ */
+trait Tables extends HasDatabaseConfigProvider[JdbcProfile] with ColumnTypeMappers {
+
+  import profile.api._
+
+  val loanTable       = TableQuery[LoanEventTable]
+  val activeLoanTable = TableQuery[ActiveLoanTable]
+  val lentObjectTable = TableQuery[LentObjectTable]
+
+  // scalastyle:off line.size.limit
+  type LoanEventRow = (
+      Option[EventId],
+      LoanType,
+      Option[JSqlTimestamp],
+      Option[ActorId],
+      Option[JSqlTimestamp],
+      MuseumId,
+      Option[EventId],
+      Option[ObjectUUID],
+      Option[CaseNumbers],
+      Option[String],
+      JsValue
+  )
+  type ActiveLoanRow = (Option[Long], MuseumId, ObjectUUID, EventId, DateTime)
+  type LentObjectRow = (Option[Long], EventId, ObjectUUID)
+  // scalastyle:on line.size.limit
+
+  class LoanEventTable(
+      val tag: Tag
+  ) extends Table[LoanEventRow](tag, Some(SchemaName), LoanEventTableName) {
+
+    val id             = column[EventId]("EVENT_ID", O.PrimaryKey, O.AutoInc)
+    val typeId         = column[LoanType]("TYPE_ID")
+    val eventDate      = column[Option[JSqlTimestamp]]("EVENT_DATE")
+    val registeredBy   = column[Option[ActorId]]("REGISTERED_BY")
+    val registeredDate = column[Option[JSqlTimestamp]]("REGISTERED_DATE")
```

Comment:
yes...I see why you've done it this way. It's the way it was done previously. If you look at the `repositories.analysis.dao.Tables` trait though, you'll see it uses `DateTime` directly. And there's a mapper for it in the `repositories.shared.dao.ColumnTypeMappers` trait.